### DISCUSSION TITLE
MEV

### DISCUSSION THREAD

**Discussion by AztecWarlord** [Cyfrin/foundry-full-course-cu#1003] [General]
I have a question. Can a MEV bot front run an arbitrage bot?

**Comment by 0xusmanf** [comment]
MEV is just a general term, it can be arbitrage, frontrunning, sandwiching, and flash loan bot.

**Comment by Ethanol48** [comment]
> I have a question. Can a MEV bot front run an arbitrage bot?


Yes it can, the MEV bot can read the transactions that you'll make, and the consequences that would result when making transactions, @PatrickAlphaC did a video in youtube experimenting with MEV bots and they even spotted a transaction made via a "proxy contract", if you can see it on chain, or the mempool the MEV bot can see it and take advantage of it.

**Comment by theirrationalone** [comment]
Based on consequences, kind of race condition between them. frontrunning: Taking advantage of info you have in advance for making your value UP ⬆, If an arbitrage bot in a cycle serving with a slower speed than a MEV bot then it's something we should be worry about. Constraints to the rescue IMO. Sorry if i went wrong ;`)

**Comment by PatrickAlphaC** [comment]
Yep - it's a big gas/validator race at that point.
